src/index.html

```diff
@@ -6,8 +7,68 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <!-- #endregion -->
+  <body class="page">
+    <div class="section page__card" data-qa="card">
+
+      <section class="section__image">
+        <img
+          class="section-image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        >
+      </section>
```

Comment:
you don't really need this section wrapper
remove it, set all paddings for card
here u have outside space near all elements, so just use paddings on card to squeeze content inside such a box
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/fcb89b2a-8665-4c18-84ba-43e855ee9cf6)


it is the reason why 40px failed, because img is 134px height, section + 4 bottom. So +40 in result we have 44 px between image and text. If you set height directly 134 to section it'll be ready to use 40px margin.
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/3c5930cd-16bf-48b3-9058-c56dda8313cf)